apolo-cli/src/apolo_cli/admin.py

```diff
@@ -1140,6 +1167,24 @@ async def add_resource_preset(
     type=int,
     help="Number of Intel GPUs",
 )
+@option(
+    "--nvidia-gpu-model",
+    metavar="GPU_MODEL_FREE_TEXT",
+    type=str,
+    help="Nvidia GPU model",
+)
+@option(
+    "--amd-gpu-model",
+    metavar="GPU_MODEL_FREE_TEXT",
+    type=str,
+    help="AMD GPU model",
+)
+@option(
+    "--intel-gpu-model",
+    metavar="GPU_MODEL_FREE_TEXT",
+    type=str,
+    help="Intel GPU model",
+)
```

Comment:
Are these options mutually exclusive or can they be used all together?

Reply:
@asvetlov they can be used all together

Reply:
In this case the formatter from https://github.com/neuro-inc/neuro-cli/pull/3101/files#diff-a031a14ac7a1382b070d1ad82b6ed4da3591f563e4726fa8a5c27b38cfca2c86R351 is not correct, it should display all available GPUs